lib/node_modules/@stdlib/stats/base/varianceyc/test/test.varianceyc.js

```diff
@@ -93,6 +131,25 @@ tape( 'if provided an `N` parameter less than or equal to `0`, the function retu
 	t.end();
 });
 
+tape( 'if provided an `N` parameter less than or equal to `0`, the function returns `NaN (accessor)`', function test( t ) {
+	var x;
+	var v;
+
+	x = toAccessorArray([ 1.0, -2.0, -4.0, 5.0, 3.0 ]);
```

Comment:
```suggestion
	x = toAccessorArray( [ 1.0, -2.0, -4.0, 5.0, 3.0 ] );
```

Same comments here and elsewhere.